package/yast2.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Jan 24 14:13:13 UTC 2018 - knut.anderssen@suse.com
+
+- Firewalld: Added methods to the wrapper class for opening ports
+  by YaPI modules (fate#323460 bsc#1071548 bsc#1076837)
```

Comment:
is it really needed? See https://fate.suse.com/323734. @kobliha ?

Reply:
That is exactly why added a very very simple wrapper, just a temporal one until it is definitely dropped.

Reply:
ok, discussed offline. YaPI is going to be marked as obsolete, but functionall, so we need to keep it working